test/extensions/filters/http/admission_control/controller_test.cc

```diff
@@ -100,6 +100,34 @@ TEST_F(ThreadLocalControllerTest, VerifyMemoryUsage) {
   EXPECT_EQ(RequestData(3, 3), tlc_.requestCounts());
 }
 
+// Test for function: averageRps
+TEST_F(ThreadLocalControllerTest, AverageRps) {
+  // Validate global_data_.requests == 0
+  tlc_.requestCounts();
+  EXPECT_EQ(0, tlc_.averageRps());
+
+  // Validate historical_data_.size() < 2
+  tlc_.recordSuccess();
+  tlc_.recordFailure();
+  EXPECT_EQ(0, tlc_.averageRps());
```

Comment:
I commented on this above, but just reiterating I'm a bit confused as to the purpose.